packages/vuetify/src/util/helpers.ts

```diff
@@ -150,6 +150,33 @@ export function addOnceEventListener (el: EventTarget, event: string, cb: () =>
   el.addEventListener(event, once, false)
 }
 
+export function addOptionsSafeEventListener (
+  el: EventTarget,
+  event: string,
+  cb: EventHandlerNonNull | (() => void),
+  options: {}
+): void {
+  const optionsSupported = Object.keys(options).reduce((obj, key) => {
+    obj[key] = false
+    return obj
+  }, Object())
+
+  try {
```

Comment:
What about caching the result so it's not evaluated every time? Of course this could take the `options` argument into account, but since it's used only for checking the `passive` option maybe we could skip the `options` argument for now

Reply:
The usage in overlayable sets `passive` to `false` while all other passive usages are setting it to `true`. Since it was going to have to take an arg for that anyway, I just went ahead and made it as flexible as possible. Good point about caching though; I'll take another look.

Reply:
I would just export it as a variable